Utilities.py

```diff
@@ -66,3 +66,6 @@ def GetFieldFromEmbed(embed, field_name):
     for field in embed_dict["fields"]:
         if field["name"].lower() == field_name.lower():
             return field["value"]
+
+def CalculatePercentage(total, acctual):
+    return "{:.2f}".format((acctual * 100) / total)
```

Comment:
```suggestion
def CalculatePercentage(total, actual):
    return "{:.2f}".format((actual * 100) / total)
```

Reply:
i did fix this before... it might have been a rebase thingy, mb

Reply:
Done